dojo/tools/securecodebox/parser.py

```diff
@@ -0,0 +1,89 @@
+import hashlib
+import json
+from urllib.parse import urlparse
+from dojo.models import Endpoint, Finding
+
+
+class SecureCodeBoxParser(object):
+    """
+    importing Findings generated by https://github.com/secureCodeBox/secureCodeBox
+    """
+
+    def get_scan_types(self):
+        return ["SecureCodeBox Findings Import"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return "SecureCodeBox Findings Import"
+
+    def get_description_for_scan_types(self, scan_type):
+        return "SecureCodeBox Findings file in JSON format can be imported."
+
+    def get_findings(self, file, test):
+        if file is None:
+            return []
+        try:
+            tree = json.load(file)
+        except json.JSONDecodeError:
+            raise ValueError("file is not valid json")
```

Comment:
You're swallowing the exception. it's better for use to have the root cause as it help us to help users.
```suggestion
        tree = json.load(file)
```

Reply:
Done